src/app/staticanalysis/modules/staticAnalysisCommon.js

```diff
@@ -442,7 +442,11 @@ function isStateVariable (name, stateVariables) {
  * @return {bool}
  */
 function isConstantFunction (node) {
-  return isFunctionDefinition(node) && node.attributes.constant === true
+  return isFunctionDefinition(node) && (
+    node.attributes.constant === true ||
+    node.attributes.stateMutability === 'view' ||
+    node.attributes.stateMutability === 'pure'
```

Comment:
I wonder why this is needed since `constant` is always set. Though for forward compatibility it should be done this way.

Reply:
That is the intention.